lib/promoted/ruby/client.rb

```diff
@@ -6,7 +6,7 @@ module Promoted
   module Ruby
     module Client
 
-      DEFAULT_DELIVERY_TIMEOUT_MILLIS = 3000
+      DEFAULT_DELIVERY_TIMEOUT_MILLIS = 250
```

Comment:
If these should be seconds, it's also fine to change the units.

Reply:
Millis is fine with the change in the Faraday setup.